.github/ISSUE_TEMPLATE/work-item-template.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Work item template
+about: This issue describes a work-item ....
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+- describe purpose of work-items
```

Comment:
```suggestion
**Describe purpose of the work-item**
Motivate briefly why we need this work item, what it will change, and what it needs to finish it.
```

Reply:
it seems you want sections rather than lists?